scripts/build/Jenkins/Jenkinsfile

```
@@ -624,10 +624,15 @@ def PreBuildCommonSteps(Map pipelineConfig, Map params, String snapshot, String
             if(env.PROJECT_OVERRIDE) {
                 external_project = env.PROJECT_OVERRIDE
             }
-            else if(params.CMAKE_LY_PROJECTS) {
+            else if(params && params.CMAKE_LY_PROJECTS) {
                 external_project = params.CMAKE_LY_PROJECTS
             }
-
+            else {
+                echo "No PROJECT_OVERIDE or CMAKE_LY_PROJECTS  defined"
+            }
+            
+            echo "external_project = ${external_project }"
```

Does groovy resolve variables with a space before the ending right curly brace?
```suggestion
            echo "external_project = ${external_project}"
```

---

It shouldn't, but good catch. This ran on o3de since this condition only triggers in a non-engine repo